### ISSUE TITLE
Conditional Logic funktioniert nicht für dieses Feld

### ISSUE THREAD

**Issue by derby** [ooksanen/acf-focuspoint#23]
Unfortunately you cannot use the Focuspoint field for ACF conditional logic.
Example:
A subsequent field should only be displayed if the Focuspoint field is filled out or not empty. Unfortunately, the Focuspoint field cannot be selected.

**Comment by ooksanen** [comment]
Hi @derby and sorry for late reply. I haven't had a chance to update this plugin for quite a while and I really don't use ACF that much anymore as native Gutenberg blocks already have similar functionality baked in.

You might want to check https://github.com/hirasso/focal-point-picker for a more up-to-date version of this field type.

**Comment by derby** [comment]
Thanks for updating! I think coding fields for custom Gutenberg blocks is not nearly as convenient as creating fields with ACF. Or do you have a smarter way?

**Comment by ooksanen** [comment]
No, not really. I just use the [@wordpress/create-block](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-create-block/) package and suitable components, such as [FocalPointPicker](https://developer.wordpress.org/block-editor/reference-guides/components/focal-point-picker/). Bit more work, but also keeps the editor UX more consistent, which I like.

You can even achieve pretty much same results as my ACF field type with built-in default blocks like cover, etc..
